pixart.js

```diff
@@ -1 +1,14 @@
+$(document).ready(function(){
 
+  $('#form').on("submit", function(event){
+    event.preventDefault();
+    $('.brush').css('background-color', $('#color-field').val());
+  })
+
+  for(var i = 0; i < 8000; i++) {
+    var square = $('<div/>').addClass('square').on('mouseover', function(){
+      $(this).css('background-color', $('#color-field').val())
```

Comment:
What you have works fine, although it might be better to pull the color form the background of the brush instead of from the input. If you pull from the input before the user has finished typing it could inadvertently try to change it to a color that it can't interpret like `blu` instead of `blue`